front-end/src/pages/Wait.jsx

```diff
@@ -1,17 +1,47 @@
 import './Wait.css';
 
+import { get } from '../utils/request';
+//import { Link } from "react-router-dom";
+
 import React from 'react';
 
-const fake_users = [{name: "Jen"}, {name: "Tanya"}, {name: "Thomas"}];
-const total = 6;
 
 function Wait() {
+  const [users, setUsers] = React.useState("")
+  const [userTotal, setUserTotal] = React.useState("?")
+  const [groupId, setName] = React.useState("b6034"); //THIS IS TEMPORARY JUST FOR NOW
+  const [friends, setFriends] = React.useState([])
+  const checkUser = async () => {
+    const users = await get(
+      '/wait',
+      {
+        groupId: groupId
+      });
+      return users;
+  };
+
+	React.useEffect(() => {
+    function initCheck() {
+      checkUser().then((response) => {
+        setUsers(response.num_users);
+        setUserTotal(response.tot_users);
+        setFriends(response.friends);
+        if (response.num_users == response.tot_users) {
```

Comment:
```suggestion
        if (response.num_users >= response.tot_users) {
```
Just in case someone has over the number of users we should redirect them anyways